app/models/manageiq/providers/vmware/manager_auth_mixin.rb

```diff
@@ -1,5 +1,6 @@
 module ManageIQ::Providers::Vmware::ManagerAuthMixin
   extend ActiveSupport::Concern
+  require 'fog/vcloud_director'
```

Comment:
Done. I put it here to make it similar to other `require_nested`, but it makes more sense to put it above.